src/previews.tm

```diff
@@ -87,46 +87,43 @@
   called without notifying the user.
 
   <\scm-code>
-    <\code>
-      (tm-define (preview-reference body body*)
```

Comment:
It seems to me that the \code environment, embedded within the scm-code, has no effect, so I removed it, but I can revert this part if I am wrong.

Reply:
It was a mistake, indeed.